runtime/common/RemoteKernelExecutor.h

```diff
@@ -85,6 +86,13 @@ class RemoteRuntimeClient
   // Return the remote capabilities of the server.
   virtual RemoteCapabilities getRemoteCapabilities() const = 0;
 
+  virtual mlir::ModuleOp lowerKernel(mlir::MLIRContext &mlirContext,
```

Comment:
nit: could we add a comment here about its functionality?

Reply:
Done